src/StreamingServer.php

```diff
@@ -171,7 +171,7 @@ public function handleConnection(ConnectionInterface $conn)
         $parser = new RequestHeaderParser($uriLocal, $uriRemote);
 
         $listener = array($parser, 'feed');
-        $parser->on('headers', function (RequestInterface $request, $bodyBuffer) use ($conn, $listener, $that) {
+        $parser->on('headers', function (ServerRequestInterface $request, $bodyBuffer) use ($conn, $listener, $that) {
```

Comment:
Good catch, but should also remove the now unneeded import? :+1:

Reply:
removed